### PR TITLE
French translation

### PR DESCRIPTION
**Pull request by Highjhacker**
## Proposed changes
Added the french translation.

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [x] Test (adding missing tests or fixing existing tests)

## Checklist

- [x] I have read the [CONTRIBUTING](https://docs.ark.io/guidebook/contribution-guidelines/contributing.html) documentation
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: test/unit/specs/i18n/i18n.spec.js [hunk 1]
```diff
@@ -10,5 +10,6 @@ describe('i18n', () => {
     expect(i18n.t('No results', 'nl')).toBe('Geen Resultaten')
     expect(i18n.t('No results', 'pt-br')).toBe('Sem resultados')
     expect(i18n.t('No results', 'pl')).toBe('Brak wyników')
+	expect(i18n.t('No results', 'fr')).toBe('Pas de résultats')
```
Comment (dated) [inline test/unit/specs/i18n/i18n.spec.js]:
```suggestion
    expect(i18n.t('No results', 'fr')).toBe('Pas de résultats')
```

**Comment by ItsANameToo** [comment]
What's up with test-node-stable, it also fails on the master branch occasionally @dated ?

**Comment by dated** [comment]
> What's up with test-node-stable, it also fails on the master branch occasionally @dated ?

yea --> #445

**Comment by ItsANameToo** [comment]
@dated thanks, that explains

(ok it technically doesn't explain, but you get it)

**Review by dated — APPROVED** [review]

**Pull request merged**